PlanarAlly/planarserver.py

```diff
@@ -411,6 +411,48 @@ def shape_wrap(player, shape):
     await sio.emit("set gridsize", grid_size, room=location.sioroom, skip_sid=sid, namespace="/planarally")
 
 
+@sio.on("new note", namespace="/planarally")
```

Comment:
This event name is not consistent with the other events you created.

This is something that I've not been consistent with myself and want to cleanup later, so this is not super important to immediately fix.